SampleApp/TangramVC.swift

```diff
@@ -8,16 +8,39 @@
 
 import UIKit
 import TangramMap
-class TangramVC:  MapViewController{
+class TangramVC:  MapViewController, MapLoadDelegate, MapSelectDelegate {
 
   override func viewDidLoad() {
     super.viewDidLoad()
-    let _ = try? loadScene("scene.yaml")
+    self.loadDelegate = self
+    self.selectDelegate = self
+    loadSceneFileAsync("scene.yaml")
   }
 
   override func viewDidAppear(_ animated: Bool) {
     super.viewDidAppear(animated)
+  }
+
+  
+  //MARK : MapLoadDelegate
+  open func mapController(_ controller: MapViewController, didLoadSceneAsync scene: String) {
     showCurrentLocation(true)
     showFindMeButon(true)
+    showTestMarker()
+  }
+  
+  //MARK : MapSelectDelegate
+  func mapController(_ mapView: MapViewController, didSelectMarker markerPickResult: TGMarkerPickResult?, atScreenPosition position: CGPoint) {
+    print("markerPicked")
+    let alert = UIAlertController(title: "Marker Selected", message: nil, preferredStyle: .alert)
+    alert.addAction(UIAlertAction(title: "Ok", style: .default, handler: nil))
+    self.present(alert, animated: true, completion: nil)
+  }
+  
+  private func showTestMarker() {
+    let markerId = markerAdd()
+    markerSetStyling(markerId, styling: "{ style: 'points', color: 'white', size: [50px, 50px], order: 2000, collide: false }")
```

Comment:
Right, `order` parameter is quite important but the value needs to be in sync with the rest of the stylesheet and within the draw rules, the documentation explains in details what it is used for https://mapzen.com/documentation/tangram/draw/#order.

To set marker orders you need to call explicitly the method `setDrawOrder`, and it's good that you mentioned that, I figured that [the method in question](https://github.com/tangrams/tangram-es/blob/6bde54513304a9e915c51decf941be10dfdc2e09/core/include/tangram/tangram.h#L252) is not exposed yet in the binary framework, this will be fixed soon.